packages/runtime/container-runtime/src/dataStore.ts

```diff
@@ -159,6 +161,7 @@ class DataStore implements IDataStore {
         private readonly logger: ITelemetryLogger,
     ) {
         this.pendingAliases = datastores.pendingAliases;
+        this.handle = fluidDataStoreChannel.handle;
```

Comment:
i would implement this by adding `Partial<IProvideFluidHandle>` to the interface, and just delegating the a getter:
```
get IFluidHandle(){return this.fluidDataStoreChannel.IFluidHandle}
```